src/Bugsnag/Client.php

```diff
@@ -33,6 +33,22 @@ public function __construct($apiKey)
     }
 
     /**
+     * Returns client config object or a single property
+     * if provided
+     *
+     * @param string|null $prop
+     * @return mixed
+     */
+    public function getConfig($prop = null)
```

Comment:
Just make that function use the same code you modified the setMetaData function too?